### PR TITLE
Add class SaltProxy for starting proxy minions

### PR DESCRIPTION
**Pull request by cro**

### CODE REVIEW

File: pytestsalt/fixtures/daemons.py [hunk 1]
```diff
@@ -629,6 +629,122 @@ def session_salt_syndic(request,
...
+
+
+@pytest.fixture
+def salt_proxy(request,
+               salt_master,
+               proxy_id,
+               proxy_config,
+               salt_proxy_before_start,  # pylint: disable=unused-argument
+               proxy_log_prefix,
+               cli_proxy_script_name,
```
Comment (s0undt3ch) [inline pytestsalt/fixtures/daemons.py]: The above `*proxy*` fixtures also need to be defined. Most of them in `pytestsalt.fixtures.config` I believe.

Comment (cro) [inline pytestsalt/fixtures/daemons.py]: OK, think I got those taken care of.

File: pytestsalt/fixtures/daemons.py [hunk 2]
```diff
@@ -1430,6 +1546,31 @@ def get_check_ports(self):
         return super(SaltMinion, self).get_check_ports()
 
 
+class SaltProxy(SaltDaemonScriptBase):
+    '''
+    Class which runs the salt-minion daemon
```
Comment (s0undt3ch) [inline pytestsalt/fixtures/daemons.py]: runs the `salt-proxy`?

File: pytestsalt/fixtures/daemons.py [hunk 3]
```diff
@@ -1430,6 +1546,31 @@ def get_check_ports(self):
...
+        script_args = ['-l', 'quiet']
+
+        script_args.extend(['--proxyid', self.config['id']])
+        if sys.platform.startswith('win') is False:
+            script_args.append('--disable-keepalive')
+        return script_args
+
+    def get_check_events(self):
+        if sys.platform.startswith('win'):
+            return super(SaltMinion, self).get_check_events()
```
Comment (s0undt3ch) [inline pytestsalt/fixtures/daemons.py]: `SaltProxy`

**Pull request merged**